Documentation/ignition.md

```diff
@@ -3,7 +3,9 @@
 
 Ignition is a system for declaratively provisioning disks from the initramfs, before systemd starts. It runs only on the first boot and handles formatting partitioning, writing files (systemd units, networkd units, dropins, regular files), and configuring users. See the Ignition [docs](https://coreos.com/ignition/docs/latest/) for details.
 
-Ignition template files can be added in the `/var/lib/bootcfg/ignition` directory or in an `ignition` subdirectory of a custom `-data-path`. Template files may contain [Go template](https://golang.org/pkg/text/template/) elements which will be evaluated with Group `metadata` and should render to JSON or YAML (which will be served by `bootcfg` as JSON).
+Normally Ignition configuration files are plain JSON, however using bootcfg you can actually use YAML which will be transformed into JSON when requested. In addition to supporting YAML for Ignition configs, bootcfg supports templating Ignition values to support more reuse and generic configurations. 
```

Comment:
How about something like?: Ignition configurations are JSON documents, but with bootcfg you may write and maintain YAML templates which will be converted to JSON when requested. Templating allows values to be substituted to facilitate reuse.